imxvpuapi/imxvpuapi_vpulib.c

```diff
@@ -2629,10 +2644,10 @@ static void imx_vpu_enc_set_mjpeg_tables(unsigned int quality_factor, EncMjpgPar
 	else
 		scale_factor = 200 - quality_factor * 2;
 
-	imx_vpu_enc_copy_quantization_table(mjpeg_params->qMatTab[DC_TABLE_INDEX0], mjpeg_enc_quantization_luma,   sizeof(mjpeg_enc_quantization_luma),   scale_factor);
```

Comment:
I studied the definition of qMatTab in the VPU API reference manual. It specifies the offsets for the Y, Cb and Cr quantization tables. I also looked at the component info tables, and as stated in the comment "The fourth byte is the index of the quantization table to use" there it is specified which qMatTab indices we are actually using. Index 1 is always specified for both Cb and Cr so we really only need to specify 0 and 1, but I kept index 2 for future reference.